README.md

```diff
@@ -10,11 +10,11 @@ home page: http://ab-initio.mit.edu/nlopt
 
 It is compiled and installed with CMake build system:
 
-    cmake .
+    cmake -DCMAKE_INSTALL_PREFIX=~/.local .
```

Comment:
@stevengj it's user-writeable (/usr/local is not), also python looks for user modules there so you don't have to set the PYTHONPATH env variable to use nlopt python bindings.